src/Firebase/Messaging/MessageData.php

```diff
@@ -34,7 +34,7 @@ public static function fromArray(array $data): self
                 );
             }
 
-            $messageData->data[(string) $key] = (string) $value;
+            $messageData->data[$key] = $value;
```

Comment:
As explained in https://github.com/kreait/firebase-php/pull/488#issuecomment-714423182, the Firebase API does want field names and values to be strings, and will convert non-strings to string when delivering the message to the clients.

Reply:
I see, I didn't know that. Thanks for the review

Reply:
For that reason I'll close PR

Reply:
Thank you nonetheless! 🌺